piecemeal-plugin/src/main/kotlin/dev/bnorm/piecemeal/plugin/fir/PiecemealFirGenerationExtension.kt

```diff
@@ -78,6 +78,12 @@ class PiecemealFirGenerationExtension(
           callableId = callableId,
         )
 
+      callableId.classId in piecemealClassIds && callableId.callableName == COPY_FUN_NAME ->
+        createFunCopy(
+          piecemealClassSymbol = owner,
+          callableId = callableId,
+        )
```

Comment:
Let's combine this branch with the previous for the `classId` check, and use a nested when for the `callableName` check.

Reply:
Done locally. Do you want me to squash the branch to a single commit before merge?